client/dashboard/sites/features.ts

```diff
@@ -1,52 +1,95 @@
 import { DotcomFeatures } from '../data/constants';
-import {
-	hasPlanFeature,
-	hasHostingFeatures,
-	hasAdvancedHostingFeatures,
-} from '../utils/site-features';
+import { hasPlanFeature } from '../utils/site-features';
 import type { Site, User } from '../data/types';
 
+export const HostingFeatures = {
+	SETTINGS_PHP: DotcomFeatures.SFTP,
+	SETTINGS_SFTP: DotcomFeatures.SFTP,
+	SETTINGS_SSH: DotcomFeatures.SSH,
+	SETTINGS_DATABASE: DotcomFeatures.SFTP,
+	SETTINGS_PRIMARY_DATA_CENTER: DotcomFeatures.SFTP,
+	SETTINGS_STATIC_FILE_404: DotcomFeatures.SFTP,
+	SETTINGS_CACHING: DotcomFeatures.ATOMIC,
+	SETTINGS_DEFENSIVE_MODE: DotcomFeatures.SFTP,
+	SETTINGS_RESTORE_PLAN_SOFTWARE: DotcomFeatures.ATOMIC,
+} as const;
+
+export type HostingFeatures = ( typeof HostingFeatures )[ keyof typeof HostingFeatures ];
+
+export function hasHostingFeature( site: Site, feature: HostingFeatures ) {
```

Comment:
I'm not sure if the current naming is ideal, as people might get confused about the difference between a hosting feature and a plan feature. The key distinction is whether the site is an atomic site (or the feature is activated). It might be better to add a comment explaining this or rename it to include terms like atomic or something like `hasActiveHostingFeature` or `IsPlanFeatureActivated`. What do you think?